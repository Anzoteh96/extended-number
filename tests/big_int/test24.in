-8854970732977040973384319858499956365138183747500412427909464352746878431
-567119591921002124706202750165977005133505932381351598093844146845597075
