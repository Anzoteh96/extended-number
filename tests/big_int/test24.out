Enter two numbers n and m:
n is -8854970732977040973384319858499956365138183747500412427909464352746878431
m is -567119591921002124706202750165977005133505932381351598093844146845597075
n + m is -9422090324898043098090522608665933370271689679881764026003308499592475506
n - m is -8287851141056038848678117108333979360004677815119060829815620205901281356
n * m is 5021827388558356548499053627955282730359911300957442367262801851531614444610687047240510414146056305916797936909248029008447720022543542634189325
n / m is 15
n % m is -348176854162009102791278606010301288135594761780138456501802150062922306
